packages/style-loader/index.js

```diff
@@ -1,10 +1,66 @@
-module.exports = ({ config }, options = {}) => config.module
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const merge = require('deepmerge');
+const { pick, mapValues } = require('lodash');
```

Comment:
We typically use ramda in place of lodash, so let's swap that out here, or use native implementations where possible.